notification-server/src/bin/service.rs

```diff
@@ -112,28 +114,39 @@ async fn main() -> anyhow::Result<()> {
     let database_connection = DatabaseConnection::create(args.db_connection).await?;
 
     let mut concordium_client = Client::new(endpoint).await?;
-    let mut receiver = concordium_client.get_finalized_blocks().await?;
-    while let Some(v) = receiver.next().await {
-        let block_hash = v?.block_hash;
-        println!("Blockhash: {:?}", block_hash);
-        let transactions = concordium_client
-            .get_block_transaction_events(block_hash)
-            .await?
-            .response;
-        for result in process(transactions).await.iter() {
-            println!("address: {}, amount: {}", result.address, result.amount);
-            for device in database_connection
-                .prepared
-                .get_devices_from_account(result.address)
+    loop {
+        info!("Reading finalized blocks");
```

Comment:
In terms of logging processing of blocks, this could be a good info message:

```
Processed block bb90584ab5f844d3c8d643f01401d1be49fa6068f048cdb4cac2e7a76dad1522 at height 18015411 with timestamp 2024-08-15 09:00:40.859 UTC. in 13ms.
```

Reply:
Timestamp is of course redundant as it is included into the log already

Reply:
These are two different timestamps. The timestamp on the left added by `tracing` is your local computer time when the log was printed while the timestamp on the right is the timestamp of the block that is processed. The timestamps are close to each other if your indexer is caught up to the top of the chain which is usually the case when running your application. So it is fine for me if you omit it for your application.

2024-08-16T08:12:42.348804Z  INFO notification_service: Processed block c8bbba2fedba1c2937aa1159c3c20fc48275b56837aeb583e7428eb332f534ee at height 18054352 with timestamp 2024-08-15 09:00:40.859 UTC

Reply:
I will write into the message which timestamp it is that we are using

Reply:
Do we have some kind of way to reach consistency on the timestamp of the logs, or how do we reach consensus?

Reply:
It is a bit of a mismatch here. In the above we are referring to blocks, however blocks do not have a timestamp attached. https://docs.rs/concordium-rust-sdk/latest/concordium_rust_sdk/v2/struct.FinalizedBlockInfo.html